Sources/NIO/ByteBuffer-core.swift

```diff
@@ -659,9 +659,15 @@ public struct ByteBuffer {
     ///
     /// - note: This method will allocate if the underlying storage is referenced by another `ByteBuffer`. Even if an
     ///         allocation is necessary this will be cheaper as the copy of the storage is elided.
-    public mutating func clear() {
```

Comment:
Can we have this function back? I’m afraid it’s a semver major to remove it. We can just implement it in terms of the new function.

Reply:
Sure! Updated!